kube_scheduler/README.md

```diff
@@ -1,66 +1,68 @@
 # Agent Check: Kubernetes Scheduler
 
+![Kube Scheduler dashboard][1]
```

Comment:
Please see the lint comment above about how to link images in this repo.

Reply:
Fixed